source/voice_control/apple.markdown

```diff
@@ -2,30 +2,56 @@
 
-## Installation
+### Prerequisites
 
-To get started make sure you're running the latest version of the [Home Assistant Companion app](https://apps.apple.com/us/app/home-assistant/id1099568401?itsct=apps_box_badge&itscg=30200) and have updated your Apple devices. Then download the two shortcuts and open them to import them:
+- [Home Assistant Companion App](https://companion.home-assistant.io/docs/getting_started/) installed
+- Have an Assistant set up: either [cloud](https://www.home-assistant.io/voice_control/voice_remote_cloud_assistant/) (recommended, more performant) or [local](https://www.home-assistant.io/voice_control/voice_remote_local_assistant/).
+- The devices you want to control with Assist are [exposed to Assist](/voice_control/voice_remote_expose_devices/)
 
-- [Assist shortcut](https://www.icloud.com/shortcuts/4172210db1d74599bf9ae4f9e1e3d6ed)
-- [Assist Button shortcut](https://www.icloud.com/shortcuts/73ebb68ca1a24ae1811b7d6971203334)
+### Starting Assist in Home Assistant
 
-![Expanded Shortcuts Action](/images/assist/ios-expanded-shortcut-action.jpeg)
+1. On your phone, open Home Assistant.
+2. In the top-right corner, select the three-dots menu and select **Assist**.
+3. [Give a command](/voice_control/custom_sentences/).
 
-There are two versions of the shortcut. The normal shortcut works better with Siri while the Button shortcut has the ability to use speech-to-text when triggered outside of Siri and can customize the input language (instead of relying on the OS language)
+### Starting Assist on your iPhone using a shortcut
 
-[_Last updated: Jan 26, 2023_](#changelog)
+This section was written using an iOS 18. Depending on your model and version, the exact steps may vary.
 
-### Adjusting the language
+To use Home Assistant Assist as your voice assistant, follow these steps:
+
+1. Create a shortcut to **Assist in App**
+2. Choose one of following options to activate Assist:
+   - [Start Assist using a back tap](#to-start-assist-using-a-back-tap).
+   - [Start Assist using the action button](#to-start-assist-using-the-action-button).
+3. [Give a command](/voice_control/custom_sentences/).
+
+#### To create a shortcut to Assist in App
+
+1. On the phone, open the **Shortcuts** app, and select **New**.
+2. Type `Home Assistant` and select **Assist in app**.
+3. **Choose** the pipeline.
+4. Select **Done**. You now have a shortcut to **Assist in app**.
+
+#### To start Assist using a back tap
+
+1. Follow the Apple documentation on [Running shortcuts by tapping the back of your iPhone](https://support.apple.com/en-gb/guide/shortcuts/apd897693606/ios) and select the shortcut to **Assist in app**.
+2. Start Assist by tapping the back of your phone.
+
+#### To start Assist using the Action button
+
+1. Go to **Settings** > **Action Button**, and scroll until you see **Controls**.
+2. Under **Home Assistant**, select **Assist**.
+3. Select your preferred pipeline.
+4. Start the Assist by holding the action button.
+
```

Comment:
Here you could mention that starting from iOS 18, Assist is also a "Control center control", which can directly be selected in the action button configuration without need to create a shortcut, it can also be used in Control center and lock screen bottom buttons

Reply:
could you rephrase as if everyone had 18? we assume the latest version.

Reply:
Sorry, I thought you have written "shortcut" but you actually already mentioned the iOS 18 control there, my bad.

For control center and lock screen:

#### To start Assist using control center
1. Open control center
2. Press and hold an empty space
3. Look for Home Assistant
4. Select Assist
5. After you see the icon in control center, tap on it again to choose which pipeline you want to use.

#### To start Assist from Lock Screen
1. Tap and hold an empty space in Lock Screen 
2. Choose one of the two bottom items where you want to replace using Assist
3. Remove the item
4. Tap on it to add a new item and find Home Assistant Assist in the list
5. After you see the icon in lock screen, tap once more to choose which pipeline you want to use.

Alternatively you can execute the same steps but add a widget below the lock screen clock.

Reply:
Something like that with your "better-wording" touch hahaha

Reply:
Wording LGTM. Added the steps.